src/components/GraphiQL.js

```diff
@@ -607,6 +609,12 @@ export class GraphiQL extends React.Component {
     }
   }
 
+  /**
+   * Runs query at a cursor and adds the `operationName`
+   * to a request if the operation defines one.
+   *
+   * @public
+   */
```

Comment:
@asiandrummer do you think this documentation is sufficient and good enough? Let me know and I can improve it